components/templates/home/_features_section.tsx

```diff
@@ -19,7 +19,6 @@ export const FeaturesSection = () => {
               alt="happy mushroom"
               width={ 200 }
               height={ 200 }
-              loading='lazy'
```

Comment:
Did you remove this on purpose?
(This was added in the commit right before this one)

Reply:
Yes! I removed the first lazy load because this should be loaded immediately on page load, with the mushroom further down loaded on a lower priority.

HOWEVER.

I have no idea if that's how this actually works. This is guessing